wasm/generateInterface.js

```diff
@@ -313,12 +313,22 @@ function generateManifest (interfaceManifest, opts) {
         call += '(get_local $callback)'
       }
 
-      call =
-        `(i64.store
-    (i32.add (get_global $sp) (i32.const ${spOffset * 32}))
-    (i64.extend_u/i32
-      ${call})))
+      if (opcode === 'CALL') {
+        call =
+          `(i64.store
+      (i32.add (get_global $sp) (i32.const ${spOffset * 32}))
+      (i64.extend_u/i32
+        (call $check_nonzero ${call}) ;; flip CALL result from EVM convention to POSIX convention 0 -> 1 and 1,2,.. -> 2
```

Comment:
This comment does not make much sense to me, is it correct? Seems that we are doing 0 -> 1, 1,2,... -> 0 with `$check_nonzero`.